overlord/snapstate/snapmgr.go

```diff
@@ -566,6 +578,15 @@ func (m *SnapManager) doClearSnapData(t *state.Task, _ *tomb.Tomb) error {
 	return nil
 }
 
+func revs(seq []*snap.SideInfo) []int {
```

Comment:
Is this used outside of the _test.go code (where it is also created). But maybe I'm slightly blind today.